pkg/base/replicated.go

```diff
@@ -444,7 +439,7 @@ func tryGetConfigFromFileContent(content []byte, log *logger.CLILogger) *kotsv1b
 	return nil
 }
 
-func getKotsKinds(u *upstreamtypes.Upstream) (*kotsutil.KotsKinds, error) {
+func getTemplatingKotsKinds(u *upstreamtypes.Upstream) (*kotsutil.KotsKinds, error) {
```

Comment:
might be good to add a comment for this indicating what "templating kots kinds" is. assuming it's the kots kinds that are needed to render template functions?

Reply:
yes, and correct.